### PR TITLE
Add Trino 441 release notes

### PR DESCRIPTION
**Pull request by colebow**
## Description

Assemble the release notes for the upcoming Trino 441 release.

## Release notes

(x) This is not user-visible or docs only and no release notes are required.

## Verification for each pull request

Format: PR/issue number, ✅ / ❌ rn ✅ / ❌ docs
✅ rn - release note added and verified, or assessed to be not necessary, set to ❌ rn before completion
✅ docs - need for docs assessed and merged, or assessed to be not necessary, set to ❌ docs before completion

## 8 Mar 2024

* #20970 ✅ rn ✅ docs
* #20986 ✅ rn ✅ docs
* #20997 ✅ rn ✅ docs

## 9 Mar 2024

* #20998 ✅ rn ✅ docs
* #20996 ✅ rn ✅ docs

## 10 Mar 2024

* #20963 ✅ rn ✅ docs
* #20990 ✅ rn ✅ docs
* #21001 ✅ rn ✅ docs

## 11 Mar 2024

* #20885 ✅ rn ✅ docs
* #21006 ✅ rn ✅ docs
* #20151 ✅ rn ✅ docs
* #21008 ✅ rn ✅ docs
* #21015 ✅ rn ✅ docs
* #21011 ✅ rn ✅ docs
* #21004 ✅ rn ✅ docs
* #21022 ✅ rn ✅ docs
* #21023 ✅ rn ✅ docs
* #21018 ✅ rn ✅ docs
* #21025 ✅ rn ✅ docs

## 12 Mar 2024

* #21003 ✅ rn ✅ docs
* #21014 ✅ rn ✅ docs
* #20276 ❌ rn ✅ docs
* #21031 ✅ rn ✅ docs
* #21013 ✅ rn ✅ docs
* #21033 ✅ rn ✅ docs
* #21040 ✅ rn ✅ docs

## 13 Mar 2024

* #21036 ✅ rn ✅ docs
* #20916 ✅ rn ✅ docs
* #21050 ✅ rn ✅ docs
* #21039 ✅ rn ✅ docs
* #21038 ✅ rn ✅ docs
* #20943 ✅ rn ✅ docs
* #20947 ✅ rn ✅ docs
* #21064 ✅ rn ✅ docs

### CODE REVIEW

File: docs/src/main/sphinx/release/release-441.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Release 441 (13 Mar 2024)
+
+## General
+
+* Fix incorrect results of window aggregations when any input data includes 
+  `NaN` or infinity. ({issue}`20946`)
+
+## Cassandra connector
+
+* Fix incorrect results when a query contains predicates on clustering columns. ({issue}`20963`)
```
Comment (mosabua) [inline docs/src/main/sphinx/release/release-441.md:10]: clustering columns ?? is that a term people understand?

File: docs/src/main/sphinx/release/release-441.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+## Hive connector
+
+* {{breaking}} Remove the `legacy` mode for the `hive.security` configuration
```
Comment (mosabua) [inline docs/src/main/sphinx/release/release-441.md:14]:
```suggestion
* {{breaking}} Remove the default `legacy` mode for the `hive.security` configuration
```

File: docs/src/main/sphinx/release/release-441.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  property, and change the default value to `allow-all`. Additionally, remove
+  the legacy `hive.allow-drop-table`, `hive.allow-rename-table`,
```
Comment (mosabua) [inline docs/src/main/sphinx/release/release-441.md:16]: maybe

the legacy properties .....

**Review by mosabua — CHANGES_REQUESTED** [review]

File: docs/src/main/sphinx/release/release-441.md (hunk 1 above)
Comment (colebow) [reply docs/src/main/sphinx/release/release-441.md:10]: I'm not sure how else I would word it

Comment (martint) [reply docs/src/main/sphinx/release/release-441.md:11]: Yes, that's a concept in Cassandra

**Pull request merged**